plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -894,10 +895,14 @@ public void testCreatePartitionedTableWithNestedTypes()
     @Test
     public void testCreatePartitionedTableAs()
     {
+        File tempDir = createTempDir();
```

Comment:
place under `getDistributedQueryRunner().getCoordinator().getBaseDataDir()` so you don't need to clean it up in the test

Reply:
✅